roles/import-validator-keys-multiclient/tasks/main.yaml

```diff
@@ -8,10 +8,60 @@
     - git
     - build-essential
     - gcc
-    - golang
+    - wget
+    - python3-pip  
+
+- name: Install pexpect
+  pip:
+    name: pexpect     
+
+- name: Find out home directory
+  command: echo $HOME
+  register: homedir
+
+- name: Checking Go is already installed
+  stat: 
+    path: "{{ homedir.stdout}}/go"
+  register: go_check
+
+- name: Remove other version of Go if installed
+  ansible.builtin.expect:
+    command: "apt-get remove --auto-remove golang-go"
+    timeout: 10
+    responses:
+       'Do you want to continue?': "y"
+  when: go_check.stat.exists and go_check.stat.isdir
+
+- name: Remove Go directory if exists
+  ansible.builtin.file:
+    path: "{{ homedir.stdout }}/go"
+    state: absent
+  when: go_check.stat.exists and go_check.stat.isdir
+
+- name: Download and Extract Go binary in '$HOME'
+  shell: wget -c https://dl.google.com/go/go1.16.8.linux-amd64.tar.gz -O - | sudo tar -xz -C "{{ homedir.stdout }}"
```

Comment:
why not use `apt install golang` or `yum install golang`?

Reply:
Ethdo will use Go version 1.6... while `apt install golang` & `yum install golang` will install Go version1.3... .